src/components/controls/textbox/Textbox.js

```diff
@@ -91,13 +92,17 @@ class Textbox extends React.Component {
     super(props);
 
     this.state = {
-      currentValue: ''
+      currentValue: props.value
```

Comment:
Heh, that's exactly what I tried first a week or two ago.  Now the textbox doesn't accept any input.